recipes/lib3to6/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set version = "202107.1047" %}
+
+package:
+  name: lib3to6
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/l/lib3to6/lib3to6-{{ version }}.tar.gz
+  sha256: 022db330b42a056a0f09207a0740445d029444890c69e20582ef99ae27acb717
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Looks like there is a console script here. Can you also add a test for this CLI below?

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
  entry_points:
    - lib3to6 = lib3to6.__main__:main
```